Games/types/Mafia/Action.js

```diff
@@ -173,6 +173,7 @@ module.exports = class MafiaAction extends Action {
   getReportsFromAlerts(alerts, player) {
     player = player || this.target;
     let reports = [];
+    const roleMessage = /role is/;
```

Comment:
Needs a more explicit regex, 

patho won't be able to get stalker's message

![image](https://github.com/UltiMafia/Ultimafia/assets/24848927/bcc15ab7-06a7-4e62-bb6d-ccb9792ef72e)